proposals/function-pointers.md

```diff
@@ -235,6 +235,23 @@ void* v = &CloseHandle;
 func* cdecl bool(IntPtr) f1 = (func* cdecl bool(IntPtr))v;
 ```
 
+### Extensible set of unmanaged calling conventions
+
+The set of unmanaged calling conventions supported by the current ECMA-335 encodings is outdated. We have seen requests to add support
+for more unmanaged calling conventions, for example:
+
+- [vectorcall](https://docs.microsoft.com/en-us/cpp/cpp/vectorcall) https://github.com/dotnet/coreclr/issues/12120
```

Comment:
~~https://docs.microsoft.com/en-us/cpp/cpp/vectorcall~~ -> https://docs.microsoft.com/cpp/cpp/vectorcall

When providing a link, the locale should be removed from the URL so when navigation occurs the current browser locale is respected.